### PR TITLE
[CARBONDATA-1139] Added testcases for CarbonHiveRecordReader

### PR DESCRIPTION
**Pull request by ramandeep30**

### CODE REVIEW

**Comment by asfgit** [comment]
Can one of the admins verify this patch?

**Comment by CarbonDataQA** [comment]
Build Success with Spark 2.1.0, Please check CI http://136.243.101.176:8080/job/ApacheCarbonPRBuilder/2268/

**Comment by asfgit** [comment]
Refer to this link for build results (access rights to CI server needed): 
https://builds.apache.org/job/carbondata-pr-spark-1.6/141/

File: integration/hive/src/test/java/org/apache/carbondata/hive/TestCarbonHiveRecordReader.java [hunk 1]
```diff
@@ -0,0 +1,687 @@
...
+
+        new MockUp<CarbonTablePath.DataFileUtil>() {
+            @Mock
+            public String getTaskNo(String carbonDataFileName) {
+                return "CarbondataFileName";
+            }
+
+        };
+
+        new MockUp<QueryModel>() {
```
Comment (cenyuhai) [inline integration/hive/src/test/java/org/apache/carbondata/hive/TestCarbonHiveRecordReader.java]: why just new a object ?

File: integration/hive/src/test/java/org/apache/carbondata/hive/TestCarbonHiveRecordReader.java [hunk 2]
```diff
@@ -0,0 +1,687 @@
...
+            @Mock
+            public AbsoluteTableIdentifier getAbsoluteTableIdentifier() {
+                CarbonTableIdentifier carbonTableIdentifier = new CarbonTableIdentifier("db", "mytable", "id");
+                return new AbsoluteTableIdentifier("", carbonTableIdentifier);
+            }
+
+            //new MockUp<QueryModel>() {
```
Comment (cenyuhai) [inline integration/hive/src/test/java/org/apache/carbondata/hive/TestCarbonHiveRecordReader.java]: please remove unused code

**Comment by cenyuhai** [comment]
Is Shivangi and anubhav100  your colleagues?

**Pull request closed without merging**